### PR TITLE
Modified existing features to comply with learnit

### PR DESCRIPTION
**Pull request by sumrdev**
I added the follow button everywhere and made it so other people cant see your exact timeline, just your cheeps. You however can see  the cheeps of who you follow

Didnt change much but feel free to suggest things

### CODE REVIEW

File: src/Chirp.Web/wwwroot/css/style.css [hunk 1]
```diff
@@ -227,3 +227,7 @@ li.topic {
   -webkit-border-radius: 5px;
   min-height: 20px;
 }
+
+.author-box.form {
+  width: min-content;
+}
```
Comment (AlbertRossJoh) [inline src/Chirp.Web/wwwroot/css/style.css:233]: EOF

**Review by AlbertRossJoh — APPROVED** [review]
Very nice!

**Review by Juules32 — CHANGES_REQUESTED** [review]
Some implementation of the authorIsThemselves seems to be missing, causing build error

File: src/Chirp.Web/CheepService.cs [hunk 2]
```diff
@@ -5,7 +5,7 @@ namespace Chirp.Web;
 public interface ICheepService
 {
     public int CheepsPerPage { get; }
-    public List<Cheep> GetCheeps(int pageNumber, string? author = null);
+    public List<Cheep> GetCheeps(int pageNumber, string? author = null, bool authorIsThemselves = false);
```
Comment (Juules32) [inline src/Chirp.Web/CheepService.cs:8]: I think we should consider naming it "isAuthor"

Comment (sumrdev) [reply src/Chirp.Web/CheepService.cs:8]: Yeah, i agree. wonder how the build got in there since mine worked, will have a look

**Review by Juules32 — APPROVED** [review]
Nice!

**Pull request merged**